### PR TITLE
IDEX-1955: fixed error when the remote branches did not disappear after remove the remote repository

### PR DESCRIPTION
**Pull request by dmytro-ndp**
Please, review fix and merge changes into the master.

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ок
what about @vparfonov ?

**Comment by vparfonov** [comment]
Ok

File: plugin-git/che-plugin-git-ext-git/src/test/java/org/eclipse/che/ide/ext/git/client/merge/MergePresenterTest.java [hunk 1]
```diff
@@ -110,25 +108,20 @@ public void testShowDialogWhenAllOperationsAreSuccessful() throws Exception {
...
-        doAnswer(new Answer() {
-            @Override
-            public Object answer(InvocationOnMock invocation) throws Throwable {
-                Object[] arguments = invocation.getArguments();
-                AsyncRequestCallback<List<Branch>> callback = (AsyncRequestCallback<List<Branch>>)arguments[2];
-                Method onSuccess = GwtReflectionUtils.getMethod(callback.getClass(), "onSuccess");
-                onSuccess.invoke(callback, branches);
-                return callback;
-            }
+        doAnswer(invocation -> {
```
Comment (vzhukovskii) [inline plugin-git/che-plugin-git-ext-git/src/test/java/org/eclipse/che/ide/ext/git/client/merge/MergePresenterTest.java]: GWT doesn't support java 8 syntax

File: plugin-git/che-plugin-git-ext-git/src/test/java/org/eclipse/che/ide/ext/git/client/merge/MergePresenterTest.java [hunk 2]
```diff
@@ -110,25 +108,20 @@ public void testShowDialogWhenAllOperationsAreSuccessful() throws Exception {
...
-            @Override
-            public Object answer(InvocationOnMock invocation) throws Throwable {
-                Object[] arguments = invocation.getArguments();
-                AsyncRequestCallback<List<Branch>> callback = (AsyncRequestCallback<List<Branch>>)arguments[2];
-                Method onSuccess = GwtReflectionUtils.getMethod(callback.getClass(), "onSuccess");
-                onSuccess.invoke(callback, branches);
-                return callback;
-            }
+
+        doAnswer(invocation -> {
```
Comment (vzhukovskii) [inline plugin-git/che-plugin-git-ext-git/src/test/java/org/eclipse/che/ide/ext/git/client/merge/MergePresenterTest.java]: GWT doesn't support java 8 syntax

**Pull request merged**